SoC/gd32vf103/Common/Include/Usb/usb_ch9_std.h

```diff
@@ -213,6 +215,17 @@ typedef struct _usb_desc_itf {
     uint8_t iInterface;                   /*!< index of the string descriptor describing the interface */
 } usb_desc_itf;
 
+typedef struct _usb_desc_itf_association {

```

Comment:
Added according to the document [USB ENGINEERING CHANGE NOTICE](http://www.skembedded.com/usb_spec/usb_20/InterfaceAssociationDescriptor_ecn.pdf).